_posts/0000-01-02-ali3ncat.md/second_slide

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
This wound sound strange to ask, but
![this](https://user-images.githubusercontent.com/78492292/107031490-03680680-6791-11eb-8fc6-2fb946b6d975.png)